connectors/migrations/db/migration_23.sql

```diff
@@ -0,0 +1,11 @@
+-- Migration created on Oct 09, 2024
+ALTER TABLE "notion_connector_block_cache_entries"
+ALTER COLUMN "childDatabaseTitle"
+DROP NOT NULL;
```

Comment:
Weird

Reply:
FWIW it was already the case:
```
childDatabaseTitle | character varying(255)   |           |          |
```

Will remove it to avoid confusion